BillingApp/BillingInfo/views.py

```diff
@@ -23,9 +23,11 @@ def post(self, request):
 
         return HttpResponseRedirect("/test_show_all/")
 
-def DeleteBilling(request, id):
-    deleted_bill = BillInfo.objects.get(id=id)
-    deleted_bill.delete()
+def DeleteBilling(request):
+    check_box_list = request.POST.getlist("check_box_list")
+    for deleted_index in check_box_list:
+        deleted_bill = BillInfo.objects.get(id=deleted_index)
+        deleted_bill.delete()
     return HttpResponseRedirect("/test_show_all/")
```

Comment:
we should use named url instead of hard code url, see this: [`reverse`](https://docs.djangoproject.com/en/3.0/ref/urlresolvers/#reverse)